server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -595,8 +591,11 @@ public EventStream getEventStream() {
          } else {
             switch (event.getConfigurationEntityType()) {
                case "cache":
+               case "template":
                   try {
-                     sse = new ServerSentEvent(eventType, cacheConfig(cacheManager, event.getConfigurationEntityName(), mediaType));
+                     Configuration config = cacheManager.getCacheConfiguration(event.getConfigurationEntityName());
+                     eventType = "create-" + event.getConfigurationEntityType();
```

Comment:
this line is not required since `eventType` is set.
```java
String eventType = event.getConfigurationEventType().toString().toLowerCase() + "-" + event.getConfigurationEntityType();
```

Reply:
Updated and I have also removed the redundant `catch` statement.